task/sendmail/0.2/sendmail.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sendmail
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
this can probably be bumped to a recent version
cc @vdemeester

Reply:
Are you referring to "  tekton.dev/pipelines.minVersion:", shouldn't it be the minimum version compatible? Does it require update?

Reply:
yes @sudhishmk

Reply:
ok, so I think need to update this version, as min compatible version wouldn't change?

Reply:
@sudhishmk, we should change this as we have bumped up API version from `v1beta1` to `v1`. `v1` was introduced in pipelines version `v0.50.x` so in that case pipelines version between `0.12.x` and `v0.50.x` won't be compatible

Reply:
Thanks for the clarification, I had checked for other tasks using v1, [which](https://github.com/tektoncd/catalog/blob/08d0a66ed819a7eadf78abee19dbb3abd4b382d4/task/shp/0.1/shp.yaml#L4) had 0.12 mentioned has min pipeline version. That's why checked.  Will update the version then squash the commits.

Reply:
@vinamra28 Done